### PR TITLE
Fix code_search_path failed in java

### PR DESCRIPTION
**Pull request by Xuxue1**
## Why are these changes needed?
fix code_search_path failed in java

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: src/ray/raylet/worker_pool.cc [hunk 1]
```diff
@@ -238,7 +238,7 @@ Process WorkerPool::StartWorkerProcess(const Language &language,
         break;
       }
       case Language::JAVA: {
-        code_search_path_str = "-Dray.job.code-search-path" + code_search_path_str;
+        code_search_path_str = "-Dray.job.code-search-path=" + code_search_path_str;
```
Comment (ffbin) [inline src/ray/raylet/worker_pool.cc:241]: How about add a test case? I am confused that it seems to be an obvious problem, but it hasn't happened before. Thanks.

**Review by ffbin — APPROVED** [review]
LGTM

**Comment by kfstorm** [comment]
@Xuxue1 May I ask what exact failure message you encountered? And is it convenient for you to add a test case?

@chaokunyang The bug seems obvious. I don't know why we didn't catch it before.

**Comment by chaokunyang** [comment]
@kfstorm This codepath only run for multi-tenancy. We need a test on multi-tenancy for it.

**Comment by kfstorm** [comment]
@chaokunyang But we already enabled multi-tenancy on the master branch.

File: src/ray/raylet/worker_pool.cc (hunk 1 above)
Comment (Xuxue1) [reply src/ray/raylet/worker_pool.cc:241]: I run a demo in ray cluster and set code-search-path in ray.conf。 but it failed.  I find start worker process command  wrong.

**Comment by Xuxue1** [comment]
@kfstorm I run a demo in ray cluster and set code-search-path in ray.conf. It can't find class.

**Comment by Xuxue1** [comment]
@kfstorm  Ok.  I try to add test cases.

**Comment by kfstorm** [comment]
@Xuxue1 Hi, any update on this issue? If you have trouble adding a test case, maybe @chaokunyang can help.

**Comment by kfstorm** [comment]
@Xuxue1 Thanks for the update. There's still a lint error. BTW, you can run `scripts/format.sh` to automatically format code.

**Review by kfstorm — APPROVED** [review]
LGTM.

**Pull request merged**